spec/lib/dispatch-rider/dispatcher_spec.rb

```diff
@@ -14,15 +14,15 @@ def process(params)
   end
 
   describe "#dispatch" do
-    let(:message){ DispatchRider::Message.new(:subject => "handle_something", :body => { :do_throw_something => true }) }
+    let(:message) { DispatchRider::Message.new(:subject => "handle_something", :body => {:do_throw_something => true}) }
 
     describe "callbacks" do
       let(:dummy) { double(:dummy) }
       let(:storage) { DispatchRider::Callbacks::Storage.new }
-      let(:message){ DispatchRider::Message.new(:subject => "handle_something", :body => { :do_throw_something => true }) }
+      let(:message) { DispatchRider::Message.new(:subject => "handle_something", :body => {:do_throw_something => true}) }
```

Comment:
Line is too long. [122/120]<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.